alacritty_terminal/src/grid/mod.rs

```diff
@@ -335,7 +332,6 @@ impl<T: GridCell + Default + PartialEq + Clone> Grid<T> {
 
         self.saved_cursor = Cursor::default();
         self.cursor = Cursor::default();
-        self.display_offset = 0;
```

Comment:
Just like to point out that this is still needed here, but `clear_history` does it for us (as it should).